hibike/unit_tests/hibike_message_tests.py

```diff
@@ -0,0 +1,386 @@
+"""
+Unit tests for functions in hibike_message.
+"""
+
+
+import unittest
+import random
+import queue
+import time
+import threading
+
+import serial
+
+import hibike_message
+import spawn_virtual_devices
+DEVICE_TYPES = list(hibike_message.DEVICES)
+
+
+def test_with_random_data(func, arg_func, kwarg_func=lambda: {}, times=5, *args, **kwargs):
+    """
+    Test FUNC with random arguments generated by ARG_FUNC and KWARG
```

Comment:
i dont like changing the case of variable names, especially in a case-sensitve language. 

also `/s/KWARG\nfunc/KWARG_FUC`